test/decl/enum/frozen.swift

```diff
@@ -10,4 +10,5 @@ internal enum Outer {
 
 @frozen @usableFromInline enum NotPublicButVersioned {} // no-warning
 
-@_frozen enum DeprecationWarning {} // expected-warning {{@frozen has no effect on non-public enums}} {{1-9=}}
```

Comment:
You lost the warning here?

Reply:
not sure why I removed that test, back now.